<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gui::MokaWindow</class>
 <widget class="QMainWindow" name="gui::MokaWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset resource="../moka_tool.qrc">
    <normaloff>:/images/moka-logo-256x256.png</normaloff>:/images/moka-logo-256x256.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMenuBar {
	background-color: qlineargradient(x1:0, y1:1, x2:0, y2:0, stop:0 #D1D1D1, stop:1 #E7E7E7);
	border-bottom-style: solid;
	border-bottom-width: 1px;
	border-bottom-color: #8A8A8A;
}

QMenuBar::item {
	spacing: 3px; /* spacing between menu bar items */
	padding: 2px 4px;
	background: transparent;
	border-top-right-radius: 2px;
	border-top-left-radius: 2px;
}

QMenuBar::item:pressed {
	background: #717171;
	color: #FFFFFF; 
	border-style: solid;
	border-width: 1px;
	border-color: #313131;
	border-bottom-style: none;
}

QMenu {
	background: #717171;
	color: #FFFFFF; 	
	border-style: solid;
	border-width: 1px;
	border-color: #313131;
}

QMenu::item:selected {
	background: #404040;
}


QMenu::item:disabled {
	color: #B7B7B7; 
	background: #717171;
}

/*
QMenu::item:disabled:selected { 
	background: transparent;
}
*/

QWidget#wdg_map_view, QWidget#wdg_molecules {
	background-color: #F5F5F5;
}

QDockWidget {
	border: 1px solid lightgray;
	color: #808080;
}

QDockWidget::title {
	text-align: left;
	background: #E8E8E8;
	/* padding-left: 5px; */
	padding: 5px;
}
</string>
  </property>
  <widget class="QWidget" name="wdg_main_widget">
   <layout class="QHBoxLayout" name="wdg_main_widget_hlayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="wdg_open_molecules_dock" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>12</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>12</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>SDF/MOL File</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#btn_open_molecules_dock {
	background-color: #FFFFFF;
	border-right: 1px solid #CECECE;
	border-style: outset;
}

QPushButton#btn_open_molecules_dock:pressed {
	background-color: #D1D1D1;
	/* border-style: inset; */
}

</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="btn_open_molecules_dock">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::TabFocus</enum>
         </property>
         <property name="text">
          <string>&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="wdg_load" native="true">
      <property name="styleSheet">
       <string notr="true">QWidget#wdg_load_controls {
	background-color: #FFFFFF;
	border-left: 1px solid #CECECE;
}

QWidget#wdg_load_panel {
	background-color: #F5F5F5;
}
</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="wdg_load_panel" native="true">
         <property name="styleSheet">
          <string notr="true">QLabel {
	background-color: transparent;
	color: #CECECE; 
	border: 3px dashed #CECECE;
	border-radius: 10px;
	/* min-width: 200px;  */
	/* min-height: 120px; */
	/* padding: 60px; */
	font: bold;
}
</string>
         </property>
         <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,0">
          <property name="verticalSpacing">
           <number>24</number>
          </property>
          <property name="margin">
           <number>6</number>
          </property>
          <item row="4" column="1">
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="2">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="1">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="lbl_load_model">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>120</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Load Model</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <property name="margin">
             <number>10</number>
            </property>
            <property name="indent">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="lbl_load_unit_info">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>120</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Load Unit Info</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <property name="margin">
             <number>10</number>
            </property>
            <property name="indent">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="lbl_load_dataset_config">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>120</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Load Dataset Configuration File</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <property name="margin">
             <number>10</number>
            </property>
            <property name="indent">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="wdg_load_controls" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>42</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>42</width>
           <height>16777215</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="margin">
           <number>6</number>
          </property>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>362</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btn_view_model">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="toolTip">
             <string>View Model</string>
            </property>
            <property name="icon">
             <iconset resource="../moka_tool.qrc">
              <normaloff>:/images/map-32x32.png</normaloff>:/images/map-32x32.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>22</width>
              <height>22</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menu_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>748</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="mnu_file">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="act_exit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="act_view_model"/>
   </widget>
   <addaction name="mnu_file"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="status_bar"/>
  <widget class="QDockWidget" name="dck_molecules">
   <property name="features">
    <set>QDockWidget::DockWidgetClosable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>SDF/MOL File</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="wdg_molecules">
    <property name="styleSheet">
     <string notr="true">QWidget#wdg_molecules_controls {
	background-color: #FFFFFF;
	border-top: 1px solid #CECECE;
}
</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QListWidget" name="lst_molecules_list"/>
     </item>
     <item>
      <widget class="QWidget" name="wdg_molecules_controls" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>3</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <property name="bottomMargin">
         <number>3</number>
        </property>
        <item>
         <widget class="QPushButton" name="btn_molecules_open">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Load SDF/MOL File</string>
          </property>
          <property name="icon">
           <iconset resource="../moka_tool.qrc">
            <normaloff>:/images/document-open-32x32.png</normaloff>:/images/document-open-32x32.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_molecules_calculate">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Calculate</string>
          </property>
          <property name="icon">
           <iconset resource="../moka_tool.qrc">
            <normaloff>:/images/system-run-32x32.png</normaloff>:/images/system-run-32x32.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="act_exit">
   <property name="icon">
    <iconset resource="../moka_tool.qrc">
     <normaloff>:/images/application-exit-32x32.png</normaloff>:/images/application-exit-32x32.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="act_view_model">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Model</string>
   </property>
   <property name="toolTip">
    <string>View Model</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../moka_tool.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>act_exit</sender>
   <signal>activated()</signal>
   <receiver>gui::MokaWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>377</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_view_model</sender>
   <signal>clicked()</signal>
   <receiver>act_view_model</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>725</x>
     <y>456</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
